include/swift/SIL/SILFunction.h

```diff
@@ -927,6 +927,9 @@ class SILFunction
   /// block inside.  This depends on there being a 'dot' and 'gv' program in
   /// your path.
   void viewCFG() const;
+  // Like the ViewCFG, but the graph does not show the basic block contents.
```

Comment:
Ok, but should be marked as SWIFT_ENABLE_TENSORFLOW and upstreamed asap :-)

Reply:
Oops, this is already up streamed. I forgot to remove this patch before sending out the PR.